docs/Users_Guide/release-notes.rst

```diff
@@ -9,148 +9,137 @@ When applicable, release notes are followed by the GitHub issue number which des
 enhancement, or new feature (`MET GitHub issues <https://github.com/dtcenter/MET/issues>`_).
 Important issues are listed **in bold** for emphasis.
 
-.. warning:: **Ensemble post-processing was added to Gen-Ens-Prod in version 10.1.0 and will be removed from Ensemble-Stat in version 11.0.0!**
+MET Version 11.0.0 release notes (20221209)
+-------------------------------------------
 
-MET Version 11.0.0-beta5 release notes (20221120)
--------------------------------------------------
-
-* Repository, build, and testing:
-
-   * MET: Enhance MET to have better signal handling for shutdown events (`dtcenter/METplus-Internal#21 <https://github.com/dtcenter/METplus-Internal/issues/21>`_).
-
-* Library Enhancements:
-
-   * Reimplement the pntnc2ascii.R utility Rscript in Python (`#2085 <https://github.com/dtcenter/MET/issues/2085>`_).
-   * **Add a Python helper script/function to transform point_data objects to met_point_data objects for Python Embedding** (`#2302 <https://github.com/dtcenter/MET/issues/2302>`_).
-
-* Application Enhancements:
-
-   * **Enhance Grid-Stat to compute SEEPS for gridded observations and write the SEEPS STAT line type** (`#1943 <https://github.com/dtcenter/MET/issues/1943>`_).
-   * **Enhance Stat-Analysis to aggregate SEEPS_MPR and SEEPS line types** (`#2339 <https://github.com/dtcenter/MET/issues/2339>`_).
-   * **Refine TCDIAG output from TC-Pairs as needed** (`#2321 <https://github.com/dtcenter/MET/issues/2321>`_).
-   * Rename the TCDIAG SOURCE column as DIAG_SOURCE (`#2337 <https://github.com/dtcenter/MET/issues/2337>`_).
-   * Enhance TC-Pairs consensus logic to compute the spread of the location, wind speed, and pressure (`#2036 <https://github.com/dtcenter/MET/issues/2036>`_).
-   * Print ASCII2NC warning message about python embedding support not being compiled (`#2277 <https://github.com/dtcenter/MET/issues/2277>`_).
-   * Eliminate Gen-Ens-Prod warning when parsing the nbhrd_prob dictionary (`#2224 <https://github.com/dtcenter/MET/issues/2224>`_).
-
-MET Version 11.0.0-beta4 release notes (20221102)
--------------------------------------------------
-
-* Repository, build, and testing:
+* Repository, build, and test:
 
+   * **Restructure the contents of the MET repository so that it matches the existing release tarfiles** (`#1920 <https://github.com/dtcenter/MET/issues/1920>`_).
+   * **Add initial files to create the MET compilation environment in the dtcenter/met-base Docker image** (`dtcenter/METbaseimage#1 <https://github.com/dtcenter/METbaseimage/issues/1>`_).
+   * Update the METbaseimage to install Python 3.8.6 from source (`dtcenter/METbaseimage#3 <https://github.com/dtcenter/METbaseimage/issues/3>`_).
```

Comment:
@JohnHalleyGotway I think we should remove this from the release notes as the issue has been reopened since the build of python 3.8.6 is not working properly yet:

Reply:
Done.